### ISSUE TITLE
Beta releases after 0.0.24.0

### ISSUE THREAD

**Issue by kartiklunkad26** [PowerShell/Win32-OpenSSH#1149]
Please answer the following

**If it is a terminal issue then please go through wiki
https://github.com/PowerShell/Win32-OpenSSH/wiki/TTY-PTY-support-in-Windows-OpenSSH**

**"OpenSSH for Windows" version**
1.0.0.0-beta, v7.6.0.0p1-Beta, v7.6.1.0p1-Beta

**Question** 

This is really a question regarding the beta tag for the last three releases. The 0.0.24.0 release is also considered non-production ready, but at least it didn't have the beta tag in the release version. 

Is there something distinctly unstable with the newer versions? 
If so, do we know when we would be releasing a new version that doesn't have a beta tag? 

Thanks,
Kartik

**Comment by jirijanata** [comment]
Hi,

usually are non-production (beta, alpha etc) versions numbered lower than 1 (with many exeptions like putty, etc). In the description is clearly written, that they wanted to synchronize future version number with OpenSSH numbering.

> Please note
> 
> to avoid confusion with various internal versions, we have made a conscious decision to synchronize future releases with the base OpenSSH software version (currently at 7.6).

This project is quite new and not every single functionality from OpenSSH is working (for example chroot). Also there are some bugs, but on the other side is OpenSSH quite stable on Windows. To your question - there is nothing distinctly unstable. To your second question - nobody knows (maybe except from manojampalam) - this is OpenSource project - there is no timeline, eta, deadlines. You can freely contribute :)

**Comment by dwatley** [comment]
I believe this "beta" ship has sailed now that OpenSSH is enabled by default and shipped with the operating system.

**Comment by bagajjal** [comment]
All the GitHub releases will be called "beta".. Windows inbox is called "production ready".

**Comment by bagajjal** [comment]
All the GitHub releases will be called as "beta" as we wouldn't be support servicing (like upgrade to new release / applying patches).
Windows inbox is called as "production ready" and we will be support servicing.

**Comment by manojampalam** [comment]
@joeyaiello  for input.

**Comment by dwatley** [comment]
I realize @bagajjal had some input on this, but maybe we're waiting on Joey's response here. I think there's some nuance to "supported" and "beta" considering from what I can tell tagged builds here are actually shipped eventually presumably without additional code changes (1803?). The cadence of that is not exactly clear. A more thorough position on this topic would certainly be appreciated.

**Comment by joeyaiello** [comment]
@dwatley: we definitely could have done a better on messaging the versioning (and we owe you a doc topic for sure on our support structure, what GitHub builds mean, etc.)

At a high-level: what ships *in* Windows is supported, and what ships on GitHub is *not* supported. 

My understanding is that once we went from `0.0.n.0` releases (which are inherently "pre-release" in that they're pre-1.0) to the official upstream OpenSSH version scheme, we appended "Beta" to denote that the 7.x releases are *not* officially/formally supported *when downloaded from GitHub*. 

So while we're still maintaining compatibility with downlevel versions of Windows where OpenSSH is not included (e.g. Win 7 or Server 2008 R2), usage on those platforms is covered only by the BSD license shipping as part of OpenSSH (i.e. use it at your own risk). 

We've thought somewhat about what it might mean to formally support downlevel platforms, but we don't have any plans there today. If you have strict requirements there, please let us know here.

**Comment by dwatley** [comment]
@joeyaiello: No, no requirements on that front. I think the support position is relatively clear. Shipped with Windows is supported, GitHub is not. To my understanding, and I imagine your doc will cover this, is that "supported" implies Microsoft's support agreements. I just take some issue with calling something "beta" and explicitly marked "unsuitable for production use" when the binaries are identical to what ships (and is used in production) to be somewhat misleading and potentially confusing. As @kartiklunkad26 mentioned: `Is there something distinctly unstable with the newer versions?`

Tangentially, I would like to point out something I've seen work well re: comms. VSCode roadmapping and development happens in the open, and there's a very clear understanding on what's prioritized and what's not. When releases can be expected, etc. I believe it would be helpful, as a consumer and contributor, to have better insight (more formalized?) into the overall lifecycle and development of OpenSSH for Windows. It would help with managing expectations.

**Comment by kartiklunkad26** [comment]
@joeyaiello Appreciate your input here. The support position absolutely makes sense. I think from the consumer of the Github releases, as @dwatley pointed that, it'd be highly appreciated if we know what stable OpenSSH release we can consume. 

From what I understand - the 7.x releases are very much stable. Would that be accurate?
